README.md

```diff
@@ -3,7 +3,7 @@
 ## Self Introduction
 My name is Yuwen Xiong. I am a second-year master student at **UChicago Harris School of Public Policy**. Below is the photo of mine:
 
-![Photo of Yuwen](C:\Users\Xyw\Documents\GitHub\CFSS\hw01\LinkedIn-profile-smaller.jpg)
+![Photo of Yuwen](LinkedIn-profile-smaller.jpg)
 
 If you want to know more about me, you can take a look at [my LinkedIn profile](https://www.linkedin.com/in/yuwenxiong/) .
 
```

Comment:
I'm also scared about finding a job.  Your interesting facts are a good icebreaker.

Reply:
Haha! Good to know!